src/test/regress/multi_schedule

```diff
@@ -52,7 +52,8 @@ test: subquery_in_targetlist subquery_in_where subquery_complex_target_list
 test: subquery_prepared_statements
 test: non_colocated_leaf_subquery_joins non_colocated_subquery_joins non_colocated_join_order
 test: cte_inline recursive_view_local_table values
-test: pg13 pg12 pg14
+test: pg13 pg12
+test: pg14
```

Comment:
can add a comment saying this should be run as a separate test since it does metadata syncing.